src/lms-harmonizer/tests_integration_sql/procs/mssql/harmonize_assignment_submissions/test_lmsassignment_submissions.py

```diff
@@ -0,0 +1,251 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Can you add a test for an update scenario?

Reply:
Yes, good catch!

Reply:
✔️